README.md

```diff
@@ -31,12 +36,6 @@ The codebase of _evmone_ is optimized to provide fast and efficient execution of
    and applied once per block during execution.
 3. Performs extensive and expensive bytecode analysis before execution.
```

Comment:
Specify we need the `advanced` flag in order to enable the Advanced interpreter (?)

Reply:
Yes, I forgot to add it in the second commit. Good now?

Reply:
Looks good.